docs/design/network-discovery/bfd-logical-port-FSM.puml

```diff
@@ -48,25 +46,26 @@ CREATING: worker-error / report error
 CREATING: worker-error / send bfd-kill
 CREATING: exit / clean active requests
 
-WAIT_STATUS --> CREATING : port-del
-WAIT_STATUS --> OPERATIONAL : online
-WAIT_STATUS --> READY : disable
-WAIT_STATUS --> REMOVING : delete
-WAIT_STATUS : enable_update / save session data
-
 OPERATIONAL --> REMOVING : session-del
-OPERATIONAL: enter / send session create
+OPERATIONAL --> HOUSEKEEPING : port-del
 OPERATIONAL: enter [have session data] / send session enable_update
 OPERATIONAL: enable_update / send session enable_update
 OPERATIONAL: disable / send session disable
-OPERATIONAL: delete / send session delete
-OPERATIONAL: port-del / send session delete
-OPERATIONAL: offline / send session offline
-OPERATIONAL: online / send session online
 
 REMOVING --> PREPARE : enable_update
 REMOVING --> [*] : port-del
 REMOVING: enter / [isOnline] send port delete request
 REMOVING: online / send port delete request
 REMOVING: port-add / send port delete request
+
+HOUSEKEEPING --> OPERATIONAL : port-add
+HOUSEKEEPING --> [*] : session-del
+HOUSEKEEPING --> PREPARE : enable_update
+HOUSEKEEPING --> DEBRIS : disable
+HOUSEKEEPING : enter / send session disable
+
+DEBRIS --> [*] : session-del
```

Comment:
Em... it does not do the cleanup. It waits for cleanup completion... There is already a `HOUSEKEEPING` state, and if we will have `CLEANUP` in addition, it will increase misunderstanding, as for me.

Do you insist on renaming?